### DISCUSSION TITLE
Unable to uninstall msipackage when application is installed in different language other than English

### DISCUSSION THREAD

**Discussion by MB266** [wixtoolset/issues#6726] [Questions]
## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Thursdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are sent to the [wix-devs](http://wixtoolset.org/documentation/mailinglist/#wix-devs) and wix-users mailing lists, as well as in Discussions.

## Discussions and support

Note that discussions are best held on the mailing lists not in issues. Please do not open issues requesting support or debugging help; first start a thread on the [wix-users](http://wixtoolset.org/documentation/mailinglist/#wix-users) mailing list and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate. You can also try out the beta [Discussions](https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions) feature.

## Instructions

Read the sections above. Delete the Triage, Discussions and support, and Instructions sections and fill out the rest.

## Bugs

If this issue is a bug:

* Which version of WiX are you building with?
v3.11
> (WiX version in major.minor.release.build format)
v3.11.2.4516
* Which version of Visual Studio are you building with (if any)?
Visual Studio 2019
> (Visual Studio version, including year and update)
Visual Studio 2019
* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
Wix Toolset Visual Studio 2019
> (WiX Toolset Visual Studio Extension version in major.minor.release.build format)

* Which version of .NET are you building with?
.NET framework 4.7.2
> (.NET version)
4.7.2
* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> (Windows version)
Windowa 10
* Describe the problem and the steps to reproduce it.
  Issue: When the application is installed in different language (ex: Spanish) using managed bootstrapper with custom UI, It will install without any errors. But when we try to uninstall the same setup file, it will uninstall only the bundle , but not MSI Package. 

We have created the MSI and supported localization  by embedding the TRANSFORMs (.mst file).  
> (Be as specific as you can and err on the side of providing too much information, including code, error messages, command lines you used to invoke the build, and so forth.)

When we install via MSI without custom bootstrapper, install and uninstall both works fine for multi languages, but same thing does not work when we embed MSI in custom bootstrapper.

* Describe the behavior you expected and how it differed from the actual behavior.

> (Be as specific as you can and err on the side of providing too m

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Reply by MB266** [reply]
Okay. Thank you.
